README.md

```diff
@@ -199,7 +204,7 @@ react-media-match provides an API for "default" queries, and a factory method to
   - `Matcher`
   - `Provider`
   - `Mock`
-  - `SSR`
+  - `ServerRender`
```

Comment:
My bad. This is left over from the very first version.
Thank you so much for stepping in and making this better